src/common/src/config.rs

```diff
@@ -273,7 +275,9 @@ mod default {
     pub fn checkpoint_interval_ms() -> u32 {
         100
     }
-
+    pub fn in_flight_barrier_nums() -> usize {
```

Comment:
default checkpoint interval ms without risingwave.toml can be changed to 250 (L275)